tmx_pico_aio/private_constants.py

```diff
@@ -132,3 +139,16 @@ class PrivateConstants:
     RED = 0
     GREEN = 1
     BLUE = 2
+
+    class SENSOR_TYPES(Enum):
```

Comment:
Rename to I2C_SENSOR_TYPES?

Reply:
No, these are any sensors, the load cell and gps don't use i2c. We might merge the sensors and modules as modules have the same capability(repeated reading)